api/redhatopenshift/HcpCluster.Management/hcpCluster-models.tsp

```diff
@@ -435,6 +435,19 @@ model NodePoolPlatformProfile {
   @visibility(Lifecycle.Read, Lifecycle.Create)
   diskStorageAccountType?: DiskStorageAccountType = "Premium_LRS";
 
+  /** The ID of the DiskEncryptionSet resource to use to encrypt the OS disks for the VMs.
+   * Can be used with either platform (Azure) managed, or customer managed encryption keys.
```

Comment:
Is this information accurate? Can you have a disk encryption set only with platform managed keys?

Reply:
Good spot. So, reading deeper you can have double encryption with platform and customer managed keys but not just platform managed. 

I'll update the string